astropy/units/quantity_helper.py

```diff
@@ -89,7 +89,7 @@ def get_converter(from_unit, to_unit):
     [None], unit ** 2 if unit is not None else dimensionless_unscaled)
 UFUNC_HELPERS[np.reciprocal] = lambda f, unit: (
     [None], unit ** -1 if unit is not None else dimensionless_unscaled)
-# cbrt only was added in numpy 1.10
+
 if isinstance(getattr(np, 'cbrt', None), np.ufunc):
```

Comment:
Here the `isinstance` check can be removed (I really should have used `not NUMPY_LT_1_10`...)